BeeAppy/src/screens/Home.js

```diff
@@ -12,6 +15,7 @@ export default class Home extends React.Component {
   };
   render() {
     const { navigate } = this.props.navigation;
```

Comment:
Can we just merge first and change that after as it's useful for checking things actually work.